openmc/model/model.py

```diff
@@ -578,7 +578,12 @@ def import_properties(self, filename):
                     cell.temperature = group['temperature'][()]
                     if self.is_initialized:
                         lib_cell = openmc.lib.cells[cell_id]
-                        lib_cell.set_temperature(group['temperature'][()])
+                        temperature = group['temperature'][()]
+                        if temperature.size > 1:
+                            for i, T in enumerate(temperature):
+                                lib_cell.set_temperature(T, i)
+                        else:
+                            lib_cell.set_temperature(temperature[0])
```

Comment:
Since `group['temperature'][()]` gets called twice here, would it be better to define `temperature` earlier like so?

```
temperature = group['temperature'][()]
cell.temperature = temperature
if self.is_initialized:
  lib_cell = openmc.lib.cells[cell_id]
  if temperature.size > 1:
    for i, T in enumerate(temperature):
      lib_cell.set_temperature(T, i)
  else:
    lib_cell.set_temperature(temperature[0])
```